tests/flow/redis_base.py

```diff
@@ -0,0 +1,70 @@
+
+import os
+import sys
+import unittest
+from redisgraph import Graph, Node, Edge
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from disposableredis import DisposableRedis
+
+from base import FlowTestsBase
+
+def redis():
+    return DisposableRedis(loadmodule=os.path.dirname(os.path.abspath(__file__)) + '/../../src/redisgraph.so')
+
+redis_server = None
+redis_cli = None
+graph = None
+
+class RedisGraphTestBase(FlowTestsBase):
+    @classmethod
+    def graphId(cls):
+        return "g"
```

Comment:
Graph ID can't be a constant, I think it should be a constructor argument.

Reply:
This is actually the common practice in the flow tests. Note that the graph is deleted when the test starts, so this is only a problem if tests run in parallel. We can use the test class name as the graph name.